src/atldld/constants.py

```diff
@@ -19,3 +19,7 @@
 
 GLOBAL_CACHE_FOLDER = pathlib.Path.home() / ".atldld"
 GLOBAL_CACHE_FOLDER.mkdir(exist_ok=True)
+
+# Volume dimensions
+REF_DIM_1UM = (13200, 8000, 11400)
+REF_DIM_25UM = (528, 320, 456)
```

Comment:
Inside of BBP yes, however, shouldn't we write things in a little bit more general way? AFAIK 10 micron volumes show up every now and then. IMO it would be nice if all the commands in the CLI (if applicable) had a parameter `--downsample-ref` (we can definitely change the name btw). Since it is one of the most fundamental things. Maybe one can argue that this `downsample_ref` could be provided for each axes separately to be as general as possible.

Reply:
Isn't this a bit too general though? I think I've mostly seen `1µm` / `10µm` / `25µm` / `100µm`.

Actually we can still define the constants for the most typical cases like 1/10/25/100, but also just calculate the new dimensions by hand in places where we want to be most general?

Reply:
Well, sure:) Feel free to do it as you want, but I think we should give user the choice (parameter in the CLI). What happens in the background (whether it's using constants or division) is of lesser importance  I guess.

Reply:
Anyway, I guess I will resolve this one for now